packages/stark-core/src/modules/session/services/session.service.ts

```diff
@@ -53,19 +53,9 @@ export const starkUnauthenticatedUserError: string = "StarkSessionService => use
 export class StarkSessionServiceImpl implements StarkSessionService {
 	public keepalive: Keepalive;
 	public session$: Observable<StarkSession>;
-	protected _fakePreAuthenticationHeaders: Map<string, string>;
+	protected _devAuthenticationHeaders: Map<string, string>;
 	public countdownStarted: boolean;
 
-	protected fakePreAuthentication: StarkPreAuthentication = {
```

Comment:
I see that now this part has been removed since it is NBB specific. That's ok but where it is now?

Normally it should be in the showcase, but it is not there...